kod/object/item/passitem/defmod/armor/robe/robedisc.kod

```diff
@@ -279,6 +288,13 @@ messages:
          Send(poOwner,@RemoveOverlay,#what=self);
       }
 
+      piItem_flags = Send(SYS, @EncodeTwoColorXLAT,
+         #color1=Send(self, @GetMasterXLATBySchool), #color2=XLAT_TO_SKIN2);
+      if IsClass(poOwner, &User) OR IsClass(poOwner, &Room)
```

Comment:
IsClass() is gross, and in this case I think unnecessary?  Just always call SomethingChanged.

Reply:
I was mostly just worried about calling functions that don't exist on certain objects, and for some reason I didn't think SomethingChanged existed on all objects.  After checking, it does so the IsClass checks are pointless.  Removed.